### ISSUE TITLE
Provide the ability to set Java options, to the JVM running Tilkynna, inside the Docker container

### ISSUE THREAD

**Issue by melissapalmer** [GrindrodBank/tilkynna#12]
Provide the ability to set Java options, to the JVM running Tilkynna, inside the Docker container

**Is your feature request related to a problem? Please describe.**
Java has the ability to set environment variables such as JMX_HOST, JMX_PORT, Memory Settings 
eg: `-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=6002 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false`
or `-Xmx3g -Xms3g`
I would like to be able to set these for the Tilkynna application inside its Docker container

**Describe the solution you'd like**
From the following posts: 
- https://medium.com/@cl4r1ty/docker-spring-boot-and-java-opts-ba381c818fa2
- https://www.veracode.com/blog/secure-development/docker-and-javaopts

By starting the jar within the Docker image with: ENTRYPOINT exec java $JAVA_OPTS and allowing JAVA_OPTS to be passed in via a Docker environment variable will solve the issue. 

**Describe alternatives you've considered**
I have attempted running the following with current docker image: 
`docker run -it --name tilkynna -p 9981:9981 -p 6002:6002 -e SPRING_DATASOURCE_URL='jdbc:postgresql://172.17.0.2:5432/tilkynna?user=postgres&password=postgres'  -e JAVA_OPTS='-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=6002 -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false' grindrodbank/tilkynna:latest`

- All starts correctly however you are still unable to use something like jconsole to get to the 6002 port. 
- if you exec onto the running container from above and run    `ps aux` you can see that the -D options have not taken effect at all: 
```bash
melissap@melissap:~$ docker exec -it tilkynna sh
/ # ls
app.jar       dev           etc           lib           media         opt           root          sbin          sys           usr
bin           drivers       home          libs-no-repo  mnt           proc          run           srv           tmp           var
/ # ps aux
PID   USER     TIME  COMMAND
    1 root      0:57 java -Djava.security.egd=file:/dev/./urandom -Dloader.path=drivers,libs-no-repo -Duser.timezone=UTC -jar /app.jar
   23 root      0:00 sh
   29 root      0:00 ps aux

```

**Comment by melissapalmer** [comment]
Starting to work on this on branch https://github.com/GrindrodBank/tilkynna/tree/12-allow-JAVA_OPTs-to-be-set

**Comment by melissapalmer** [comment]
This has been resolved with PR #13
